server/consumer.go

```diff
@@ -69,6 +70,11 @@ type ConsumerConfig struct {
 	Direct bool `json:"direct,omitempty"`
 }
 
+type PushActive struct {
+	Subject string `json:"subject,omitempty"`
```

Comment:
I think subject will always be present, but that's ok.

Reply:
Yes agree, will update and if local passes will merge.